### ISSUE TITLE
Using ES6 breaks testing

### ISSUE THREAD

**Issue by DanMMX** [DaftMonk/generator-angular-fullstack#1081]
I changed all the client code to ES6 with the `babel` option enabled on the `canary` branch.

After changing the following file:

```
'use strict';

angular.module('appName')
  .directive('footer', () => {
    return {
      templateUrl: 'components/footer/footer.html',
      restrict: 'E',
      link: (scope, element) => {
        element.addClass('footer');
      }
    };
  });

```

I'm getting this error when doing `grunt test`

```
INFO [PhantomJS 1.9.8 (Mac OS X 0.0.0)]: Connected on socket DuFkY1bEL9QB0nekjB20 with id 21853607
PhantomJS 1.9.8 (Mac OS X 0.0.0) ERROR
  SyntaxError: Parse error
  at /Users/dan/Documents/projects/centro/client/components/footer/footer.directive.js:4

PhantomJS 1.9.8 (Mac OS X 0.0.0): Executed 0 of 0 ERROR (0.095 secs / 0 secs)

Warning: Task "karma:unit" failed. Use --force to continue.

Aborted due to warnings.
```

**Comment by kingcody** [comment]
@DanMMX thank you very much for the bug report.

**Comment by DanMMX** [comment]
:+1: thanks for guiding me to fix it, wasn't trivial for me that's why I didn't made a PR, but was an easy one

**Comment by kingcody** [comment]
closed by #1084
